src/client/views/components/todoItem.jsx

```diff
@@ -116,6 +118,17 @@ export default React.createClass({
         });
     },
 
+    makeElementFullyVisible: function () {
```

Comment:
What do you think about calling this `scrollToElement`? Or something to that effect?